packages/nimble-components/src/table/tests/table-sorting.spec.ts

```diff
@@ -733,4 +733,165 @@ describe('Table sorting', () => {
             expect(getRenderedRecordIds()).toEqual(['3', '4', '1', '2']);
         });
     });
+
+    describe('interactive sorting', () => {
+        const data: readonly SimpleTableRecord[] = [
+            {
+                id: '1',
+                stringData1: 'a1',
+                stringData2: 'b1',
+                stringData3: 'c1'
+            },
+            {
+                id: '2',
+                stringData1: 'a2',
+                stringData2: 'b2',
+                stringData3: 'c2'
+            },
+            {
+                id: '3',
+                stringData1: 'a3',
+                stringData2: 'b3',
+                stringData3: 'c3'
+            },
+            { id: '4', stringData1: 'a4', stringData2: 'b4', stringData3: 'c4' }
+        ] as const;
+
+        async function shiftClickOnElement(
+            elementToClick: HTMLElement
+        ): Promise<void> {
+            const clickEvent = new MouseEvent('click', {
+                shiftKey: true,
+                bubbles: true
+            } as MouseEventInit);
+            elementToClick.dispatchEvent(clickEvent);
+            await waitForUpdatesAsync();
+        }
+
+        it('does not affect sortIndex / sortDirection', async () => {
```

Comment:
nit: I was initially confused by the test name since I thought interactive sorting did affect the index/direction of the column being sorted. How about this?
```suggestion
        it('does not affect sortIndex / sortDirection of other column', async () => {
```

Would it make selse to also verify that `currentSortIndex` and `currentSortDirection` are cleared on `column1` even though `sortIndex` and `sortDirection` are preserved?

Reply:
I think this test is OK as-is. It's checking that interactive sorting doesn't update sortIndex/sortDirection of the sorted column (it only affects currentSortIndex/currentSortDirection). The direction cycling tests check that the currentSortIndex/Direction properties get the correct values.